src/jlapi.c

```diff
@@ -352,6 +352,37 @@ JL_DLLEXPORT void (jl_cpu_wake)(void)
     jl_cpu_wake();
 }
 
+#ifndef _OS_WINDOWS_
+#  define UNW_LOCAL_ONLY
+#  include <libunwind.h>
+#define STRINGIFY1(x) #x
+#define STRINGIFY2(x) STRINGIFY1(x)
+#if defined(_OS_LINUX_)
+#if defined(_CPU_X86_64_)
+asm("_jl_unw_getcontext: movq " STRINGIFY2(unw_tdep_getcontext) "@GOTPCREL(%rip), %rax\n"
+    "jmpq *%rax");
+#elif defined(_CPU_X86_)
+asm("_jl_unw_getcontext: movq " STRINGIFY2(unw_tdep_getcontext) "@GOTPCREL(%eip), %eax\n"
+    "jmpq *%eax");
```

Comment:
And it turns out that on AArch64 `unw_tdep_getcontext` is a function macro too.....

FWIW, this is how this should work on aarch64, if you are going to do another rebase.
```diff
diff --git a/src/jlapi.c b/src/jlapi.c
index 384de02..cd98f6a 100644
--- a/src/jlapi.c
+++ b/src/jlapi.c
@@ -364,6 +364,8 @@ asm("_jl_unw_getcontext: movq " STRINGIFY2(unw_tdep_getcontext) "@GOTPCREL(%rip)
 #elif defined(_CPU_X86_)
 asm("_jl_unw_getcontext: movq " STRINGIFY2(unw_tdep_getcontext) "@GOTPCREL(%eip), %eax\n"
     "jmpq *%eax");
+#elif defined(_CPU_AARCH64_)
+asm("_jl_unw_getcontext: b getcontext");
 #endif
 #endif
```
I verified in gdb that it branches to the same plt slot with `rec_backtrace`.